boards/xtensa/heltec_wireless_stick_lite_v3/heltec_wireless_stick_lite_v3.yaml

```diff
@@ -0,0 +1,23 @@
+identifier: heltec_wireless_stick_lite_v3
+name: Heltec Wireless Stick Lite (V3)
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - gpio
+  - uart
+  - i2c
+  - spi
+  - can
+  - counter
+  - watchdog
+  - entropy
+  - pwm
+  - dma
+  - lora
+testing:
+  ignore_tags:
+    - net
+    - bluetooth
+    - lora
```

Comment:
Any reason to remove `lora` from the supported features?

Reply:
I was able to test the LoRa samples using two boards, one with the receive firmware and another with the send firmware. The latter works but the former blocks and doesn't report receiving any message.

Reply:
Hi @sylvioalves, I have repeated the test using the latest SDK and the master branch as of recently. It appears to work now --the receive firmware--, but the output is garbled, so maybe I need to tweak the config parameters for the radio (US902-928 frequency)?

```bash
*** Booting Zephyr OS build v3.5.0-rc1-170-g7fb2929e5ba0 ***
[00:00:00.297,000] <inf> lora_receive: Synchronous reception
?_?H??&??9Zi? (RSSI:0dBm, SNR:-19dBm): Received data: ??p?t?T???????H'?S?
```

I think the issue was fixed by the PR by @fabiobaltieri referenced in https://github.com/zephyrproject-rtos/zephyr/issues/62427